core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -253,6 +255,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/InteractionObjectFactory.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/AnswerGroupObjectFactory.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/FallbackObjectFactory.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/HintObjectFactory.js"></script>
```

Comment:
Remember to include these in the collection editor/player pages as well--at the moment, if you don't it'll cause errors (I think some of your failing tests are from this). Hopefully this will eventually be decoupled so we don't always have to update both